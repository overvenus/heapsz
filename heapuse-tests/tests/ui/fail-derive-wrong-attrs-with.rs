use heapuse_derive::Heap;
use heapuse_example::*;

#[derive(Heap)]
pub struct WrongAttr {
    #[heap_size(with)]
    pub a: primitive::StructPrimitives,
}

fn main() {}
