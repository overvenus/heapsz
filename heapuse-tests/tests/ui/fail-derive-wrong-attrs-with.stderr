error: heap_size attribute `with` must be followed by a module path, `with = "some::mod"`
 --> tests/ui/fail-derive-wrong-attrs-with.rs:6:17
  |
6 |     #[heap_size(with)]
  |                 ^^^^
